extensions/authentication-jwt/src/keys.ts

```diff
@@ -34,3 +35,49 @@ export namespace UserServiceBindings {
   export const USER_CREDENTIALS_REPOSITORY =
     'repositories.UserCredentialsRepository';
 }
+
+/**
+ * Constant values used when generating refresh token.
+ */
+export namespace RefreshTokenConstants {
+  /**
+   * The default secret used when generating refresh token.
+   */
+  export const REFRESH_SECRET_VALUE = 'r3fr35htok3n';
+  /**
+   * The default expiration time for refresh token.
+   */
+  export const REFRESH_EXPIRES_IN_VALUE = '216000';
+  /**
+   * The default issure used when generating refresh token.
+   */
+  export const REFRESH_ISSURE_VALUE = 'loopback4';
+}
+
+/**
+ * Bindings related to token refresh service. The omitted explanation can be
+ * found in namespace `RefreshTokenConstants`.
+ */
+export namespace RefreshTokenServiceBindings {
+  export const REFRESH_TOKEN_SERVICE = BindingKey.create<RefreshTokenService>(
+    'services.authentication.jwt.refresh.tokenservice',
+  );
+  export const REFRESH_SECRET = BindingKey.create<string>(
+    'authentication.jwt.refresh.secret',
+  );
+  export const REFRESH_EXPIRES_IN = BindingKey.create<string>(
+    'authentication.jwt.referesh.expires.in.seconds',
```

Comment:
Typo? Are instances of 'referesh' meant to be 'refresh'?

Reply:
thanks @dougal83  for spoting out typos.